c7n/resources/rdsparamgroup.py

```diff
@@ -80,6 +81,118 @@ def get_pg_name(self, pg):
         return pg['DBClusterParameterGroupName']
 
 
+class ParameterFilter(ValueFilter):
+    """
+    Applies value type filter on db parameter values.
+
+    """
+
+    schema = type_schema('db-parameter', rinherit=ValueFilter.schema)
+    schema_alias = False
+    permissions = ('rds:DescribeDBParameters',)
+    policy_annotation = 'c7n:MatchedDBParameter'
+
+    @staticmethod
+    def recast(val, datatype):
+        """ Re-cast the value based upon an AWS supplied datatype
```

Comment:
I would prefer not to see this duplicated between rds and rds cluster modules, ie. its a static method in the original so you can omit this and just import rds module and use the ParameterFilter.recast method at call sites instead of copy/pasting here.